core/src/main/scala/org/apache/spark/streaming/eventhubs/EventHubsClientWrapper.scala

```diff
@@ -131,22 +132,27 @@ private[eventhubs] class EventHubsClientWrapper extends Serializable with EventH
     // Create Eventhubs client
     eventhubsClient = EventHubClient.createFromConnectionStringSync(connectionString)
 
+    // Enabling flag to receive runtime info from eventhub client.
+    val receiverOptions = new ReceiverOptions
+    receiverOptions.setReceiverRuntimeMetricEnabled(true)
```

Comment:
https://github.com/Azure/azure-event-hubs-java/blob/b378b50b25253c036dad62cffbebe490f9bcfe15/azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/ReceiverRuntimeInformation.java#L17, the last sequence number will be 64 bytes....so I think it is too expensive to introduce as a temporal solution

I do not know their plan, maybe soon, since they mentioned this feature to me several weeks ago